google-ads-examples/src/main/java/com/google/ads/googleads/examples/hotelads/AddHotelCallout.java

```diff
@@ -0,0 +1,231 @@
+// Copyright 2018 Google LLC
```

Comment:
Can we call this AddHotelCalloutExtension.java? This would be consistent with other examples, eg. [AddGoogleMyBusinessLocationExtensions](https://github.com/googleads/google-ads-java/blob/master/google-ads-examples/src/main/java/com/google/ads/googleads/examples/extensions/AddGoogleMyBusinessLocationExtensions.java)

Reply:
+1. I think this would fit better in the package "extensions" too.

Reply:
Moved the file and renamed.